src/Compilers/CSharp/Portable/Emitter/Model/PEModuleBuilder.cs

```diff
@@ -914,6 +914,9 @@ internal static Cci.IGenericParameterReference Translate(TypeParameterSymbol par
 
                 case SymbolKind.TypeParameter:
                     return Translate((TypeParameterSymbol)typeSymbol);
+
+                case SymbolKind.TupleType:
+                    return Translate(((TupleTypeSymbol)typeSymbol).UnderlyingTupleType, syntaxNodeOpt, diagnostics);
```

Comment:
`// PROTOTYPE: need to write metadata for cross-assembly visibility of tuple member names`